core/src/main/java/org/jdbi/v3/core/CloseException.java

```diff
@@ -11,13 +11,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jdbi.v3.core.exception;
+package org.jdbi.v3.core;
 
-public class UnableToCloseResourceException extends JdbiException
+/**
+ * Thrown when an exception occurs while attempting to close a database resource.
+ */
+public class CloseException extends JdbiException
 {
     private static final long serialVersionUID = 1L;
 
-    public UnableToCloseResourceException(String string, Throwable throwable)
+    public CloseException(String string, Throwable throwable)
```

Comment:
This was something I wanted to try: shortening our exception names. If we made them more broad we could reuse the same exceptions for some things, and then the names don't have to be verbose--just the exception message.